### PR TITLE
Fix pfri export to pandas, fixes #35

### LINKED ISSUE #35: pfri.export_to_pandas() bug
pfri.export_to_pandas() does not always work, because sometimes wrong ElmRes is taken.

### PR DESCRIPTION
**Pull request by bertrtam00**

### CODE REVIEW

File: src/powfacpy/base_interface.py [hunk 1]
```diff
@@ -688,23 +694,20 @@ def get_project_settings(self):
...
-    comres.pResult = None # export only selected variables
     # Insert time
     time_variable_name = powfacpy.PFResultsInterface._get_time_variable_name_from_elmres(elmres)
     
     first_column_is_time = results_variables_lists['variables'][0] == time_variable_name
     if not first_column_is_time: # add time as first column
-      results_variables_lists['result_objects'].insert(0, results_variables_lists['result_objects'][0])
-      results_variables_lists['elements'].insert(0, results_variables_lists['result_objects'][0])
+      results_variables_lists['result_objects'].insert(0, elmres)
+      results_variables_lists['elements'].insert(0, elmres)
```
Comment (Sciemon) [inline src/powfacpy/base_interface.py:710]: This is to move all items by one index to align with the 'variables' list where the time variable must be added?

File: src/powfacpy/results_interface.py [hunk 2]
```diff
@@ -38,10 +38,10 @@ def export_to_pandas(self, result_objects, elements, variables):
         variables: List of variable names for each result variable
     """
     FILE_NAME = 'temp'
-    FILE_PATH = getcwd()
-    FULL_PATH = FILE_PATH + "\\" + FILE_NAME + ".csv"
+    file_path = getcwd()
```
Comment (Sciemon) [inline src/powfacpy/results_interface.py:41]: file_path and full_path are still constants and could be written in capital letters?

File: src/powfacpy/results_interface.py [hunk 3]
```diff
@@ -70,7 +70,10 @@ def _get_time_variable_name_from_elmres(elmres):
       29:'b:ucttime' # quaidynamic simulation
       # TODO to be continued if needed
     }
-    return time_names[simulation_type_number]
+    try:
+      return time_names[simulation_type_number]
+    except KeyError:
+      print('PF simulation type number {} not known or not implemented yet. Maybe to be added to time_names dictionary?')
```
Comment (Sciemon) [inline src/powfacpy/results_interface.py:76]: This will not throw an exception if a KeyError is raised. 
Should it be an f-string with "{elmres.calTp}"?

**Review by Sciemon — APPROVED** [review]

File: src/powfacpy/results_interface.py (hunk 3 above)
Comment (bertrtam00) [reply src/powfacpy/results_interface.py:76]: I forgot to include, I'll do a fix. 
Do you think it's fine like this? 

```
    try:
      return time_names[simulation_type_number]
    except KeyError:
      print('PF simulation type number {} not known or not implemented yet. Maybe to be added to time_names dictionary?'.format(simulation_type_number))
```

**Pull request merged**